dfx.json

```diff
@@ -14,7 +14,7 @@
   },
   "networks": {
     "local": {
-      "bind": "127.0.0.1:8000",
+      "bind": "0.0.0.0:8000",
```

Comment:
What's the issue with 127.0.0.1? Guessing this may have something to do with being accessible from Docker. If that's the case, can you create a ticket about reverting this once/if we find a better setup?

Reply:
Yes, excactly. The requests coming out of docker do apparently not use `127.0.0.1`. This will be reverted once we use docker-compose.